3 - Assignment/240714 - Rust Basic + Linux Kernel, Week 2, Day 2/collatz-sequence/src/lib.rs

```diff
@@ -1,6 +1,15 @@
 /// Determine the length of the collatz sequence beginning at `n`.
 pub fn collatz_length(mut n: i32) -> u32 {
-    todo!("Implement this")
+    let mut length = 1_u32;
+    while n != 1 {
+        if n % 2 == 0 {
+            n /= 2;
+        } else {
+            n = 3 * n + 1;
+        }
```

Comment:
이렇게 축약형으로 쓸 수도 있습니다.
```suggestion
        n = if n % 2 == 0 { n / 2 } else { 3 * n + 1 };
```